intelmq/bots/collectors/tcp/collector.py

```diff
@@ -54,7 +54,7 @@ def process(self):
                 if msg:  # if the partner connection ended, our message are already sent
                     conn.sendall(b"Ok")
                     pass
-        except socket.error:
+        except OSError:
```

Comment:
socket.error has been removed in favor of OSError?

Reply:
https://docs.python.org/3/library/socket.html?highlight=socket%20error#socket.error
> A deprecated alias of OSError.